2-HT-OOP/src/Sword.ts

```diff
@@ -0,0 +1,23 @@
+// your code goes here
+import { Weapon } from "./Weapon";
+
+export class Sword extends Weapon{
+   
+    constructor(
+        value:number,
+        weight:number,
+        baseDamage:number,
+        baseDurability:number,
+    ){   
+        super("Sword",weight,value)
+        this.BaseDurability = baseDurability;
+        this.BaseDamage = baseDamage;
+        
+    }
+
+    public polish():void{
+        this.Modifier_change_rate+=this.Modifier_change_rate;
+        /* How to increase the base damage by 25% ?? */
```

Comment:
In the above statement, you can multiple the `Modifier_change_rate` with 0.25 to get 25%. And you also need to handle a case where it's not exceeding the 1.